docs/flex/interactive/development/stored_procedure/cpp_procedure.md

```diff
@@ -171,8 +171,106 @@ For more tails about Java SDK Interface, please refer to [Java SDK Procedure API
 
 #### gsctl 
 
-TODO
+In addition to registering stored procedures via SDKs, you can also register C++ procedures using `gsctl`. With Interactive deployed, you can register a C++ stored procedure similarly to [creating a Cypher stored procedure](../../stored_procedures.md).
```

Comment:
I suggest moving gsctl method before calling from python and java sdk, as this is, in my opinion, the default way of specifying stored procedure.